client/src/Sites/Sites.js

```diff
@@ -175,106 +175,115 @@ function Sites() {
         </Link>
       )}
       <div className="filter">
-        <div className="filter-text">Filter</div>
-        <div className="filter-button">
-          <button
-            class="btn filter-button dropdown-toggle"
-            type="button"
-            id="dropdownMenuButton"
-            data-toggle="dropdown"
-            aria-haspopup="true"
-            aria-expanded="false">
-            Nutrition Partner
-          </button>
-          <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-            {partners?.map((partner) => (
-              <div onClick={() => onFilterClick('nutritionPartnerId', partner.id)} class="dropdown-item">
-                {filters.nutritionPartnerId === partner.id && <span>*</span>}
-                {partner.name}
-              </div>
-            ))}
+        <div className="filter-options">
+          <div className="filter-text">Filter</div>
+          <div className="filter-button">
+            <button
+              class="btn filter-button dropdown-toggle"
+              type="button"
+              id="dropdownMenuButton"
+              data-toggle="dropdown"
+              aria-haspopup="true"
+              aria-expanded="false"
+              style={filters.nutritionPartnerId && { border: '1px solid #037493' }}>
```

Comment:
You typically don't want to use inline styles like this... instead, create a new modifier css class name to add...

There's a library called classnames that helps dynamically generate the class list for an element. Note that this file has a bunch of class attributes that haven't been converted to className for React yet...